react-components/hello-world.md

```diff
@@ -0,0 +1,386 @@
+
+## Inspecting the Code
+
+Our project should have the following structure:
+
+```
+├── public
+│   └── index.html
+├── src
+│   ├── App.css
+│   ├── App.js
+│   ├── App.test.js
+│   ├── index.css
+│   ├── index.js
+│   ├── logo.svg
+│   ├── reportWebVitals.js
+│   └── setupTests.js
+├── package.json
+├── README.md
+└── yarn.lock
+```
+
+In this curriculum, we will intentionally not cover the following files:
+
+- The contents of `public` besides `index.html`
+- `logo.svg`
+- `reportWebVitals.js`
+- `setupTests.js`
+
+There are two files that are the most important to understand when developing React for the first time:
+
+1. `src/index.js`
+1. `src/App.js`
+
+### Exploring `index.js`
+
+Let's go ahead and open this project up in our text editor to examine the files that have been created for us. We'll begin with `src/index.js` and practice our reading skills.
+
+```js
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById("root")
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
+```
+
+<!-- Question 2 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: checkbox
+* id: e447ec71
+* title: Hello, World!
+##### !question
+
+`src/index.js` imports several files. Select the files that are imported.
+
+##### !end-question
+##### !options
+
+* `React`
+* `ReactDOM`
+* `'./index.css'`
+* `App` from `'./App'`
+* `logo` from `'./logo.svg'`
+* `'./index.js'`
+
+##### !end-options
+##### !answer
+
+* `React`
+* `ReactDOM`
+* `'./index.css'`
+* `App` from `'./App'`
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 3 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: 855add45
+* title: Hello, World!
+##### !question
+
+`ReactDOM.render( ... );` is a...
+
+##### !end-question
+##### !options
+
+* variable
+* function call
+* class
+
+##### !end-options
+##### !answer
+
+* function call
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
+<!-- Question 4 -->
+<!-- prettier-ignore-start -->
+### !challenge
+* type: multiple-choice
+* id: b56db597
+* title: Hello, World!
+##### !question
+
+The following code snippet is in `src/index.js`.
+
+```js
+<React.StrictMode>
+  <App />
+</React.StrictMode>
+```
+
+This code is...
+
+##### !end-question
+##### !options
+
+* An import statement
+* A comment in React
+* The first argument when calling `ReactDOM.render()`
+* An object literal
+
+##### !end-options
+##### !answer
+
+* The first argument when calling `ReactDOM.render()`
+
+##### !end-answer
+### !end-challenge
+<!-- prettier-ignore-end -->
+
```

Comment:
this looked like a really bare transition from the questions back to the text. Some kind of header might help.
```suggestion

### `index.js` Is the Starting Point

```